pages/articles/page/[page].tsx

```diff
@@ -29,15 +30,30 @@ export const getStaticProps: GetStaticProps<Props> = async context => {
   const pageURLParameter = context.params?.page;
   const pageNumber = !pageURLParameter || isNaN(+pageURLParameter) ? 1 : +pageURLParameter;
 
+  if(pageNumber < 0){
+    return notFoundRedirect;
+  }
+
+  if (pageNumber === 1 || pageNumber === 0) {
+    return {
+      redirect: {
+        destination: '/articles',
```

Comment:
Could we load this from some constant (from routing)?

Reply:
Well the if I put "articles" in pageCodenames it is trying to generate sites in src/[slug.tsx] and there is a conflict then

![image](https://github.com/kontent-ai-bot/kontent-ai-new-sample-app/assets/54802833/d150afe9-a1a0-4790-b06d-567c79784845)

this is in /src/[slug.tsx]
![image](https://github.com/kontent-ai-bot/kontent-ai-new-sample-app/assets/54802833/74cb54f3-1f55-44f8-a698-166682522512)

I am not sure why it is like that and I think it should be fixed, however, I would make another task for that to not block taxonomies.